app/src/main/java/de/westnordost/streetcomplete/util/ktx/Resources.kt

```diff
@@ -10,13 +10,20 @@ import com.charleskorn.kaml.decodeFromStream
 import de.westnordost.streetcomplete.view.DrawableImage
 import de.westnordost.streetcomplete.view.Image
 import de.westnordost.streetcomplete.view.ResImage
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
 
 inline fun <reified T> Resources.getYamlObject(@RawRes id: Int): T =
     Yaml.default.decodeFromStream(openRawResource(id))
 
 /** shortcut for [getYamlObject] with included type information */
 fun Resources.getYamlStringMap(@RawRes id: Int): Map<String, String> = this.getYamlObject(id)
 
+@OptIn(ExperimentalSerializationApi::class)
```

Comment:
what's experimental here?

Reply:
The `Json.decodeFromStream(stream)` function that infers the deserializer from the output type instead of an explicit parameter.